src/components/organisms/ChatSection/ChatSection.tsx

```diff
@@ -1,9 +1,36 @@
 import { Button, Box, Typography, TextField } from "@mui/material";
 import { Send as SendIcon } from "@mui/icons-material";
-import { SentMessage } from "components";
 import styles from "./ChatSection.module.css";
+import { SentMessagesList } from "components";
+import React, { useState } from "react";
 
 export const ChatSection = () => {
+  const [messageToSend, setMessageToSend] = useState<string>("");
+
+  const tempMessage1 = {
+    messageText: "Wiadomosc 1 TEST",
+    isOwn: false,
+    isLast: false,
+  };
+  const tempMessage2 = {
+    messageText: "Wiadomosc 2 TEST",
+    isOwn: false,
+    isLast: false,
+  };
+  const tempMessage3 = {
+    messageText: "Wiadomosc 3 TEST",
+    isOwn: false,
+    isLast: false,
+  };
+
+  const tempMessages = [tempMessage1, tempMessage2, tempMessage3];
```

Comment:
Może możemy wyeksportować do zewnętrznego pliku?

Reply:
możemy, ale to na chwilowe testy :) niebawem zniknie